app/models/like.rb

```diff
@@ -1,4 +1,7 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, foreign_key: :user_id, class_name: 'User'
   belongs_to :post
+  def update_likes_counter
```

Comment:
- [ ] [OPTIONAL] In order to have a more standard and professionally well-written code, I will suggest you leave/create a space between the `belong_to` relationship and the `update_likes_counter` method.